components/tikv_util/src/time.rs

```diff
@@ -29,9 +29,17 @@ pub fn duration_to_sec(d: Duration) -> f64 {
     d.as_secs() as f64 + (nanos / 1_000_000_000.0)
 }
 
+/// Converts Duration to microseconds.
+#[inline]
+pub fn duration_to_us(d: Duration) -> u64 {
+    let nanos = u64::from(d.subsec_nanos());
+    // Most of case, we can't have so large Duration, so here just panic if overflow now.
+    d.as_secs() * 1_000_000 + (nanos / 1_000)
```

Comment:
Actually, when compile in release mode, integer overflow will cause wrap around silently instead of panic.

Reply:
Update the comments